game_class.py

```diff
@@ -0,0 +1,39 @@
+"""
+    Create an abstract class
```

Comment:
Purpose:
Add a . at the end of each sentence.
Start each sentence with a capital letter.